src/config.rs

```diff
@@ -135,6 +135,9 @@ pub struct GeneralConfig {
 
     #[serde(default = "def_half")]
     pub xr_alt_click_sensitivity_release: f32,
+
+    #[serde(default = "def_true")]
+    pub screens_slideable: bool,
```

Comment:
this applies to not only screens, so let's call it something else
`allow_sliding` or something, maybe. other than that, i'm done complaining :P